scene/2d/skeleton_2d.cpp

```diff
@@ -153,16 +154,6 @@ void Bone2D::_notification(int p_what) {
 			}
 #endif // TOOLS_ENABLED
 		} break;
-
-		case NOTIFICATION_MOVED_IN_PARENT: {
-			if (skeleton) {
-				skeleton->_make_bone_setup_dirty();
-			}
-			if (copy_transform_to_cache) {
-				cache_transform = get_transform();
-			}
```

Comment:
This part seems to be lost, not sure if it's necessary, but it's not part of `_make_bone_setup_dirty`.

Reply:
It looks kind of useless to me, can't see how changing order can affect the transform.